h2o-genmodel/src/main/java/hex/genmodel/algos/gbm/GbmMojoModel.java

```diff
@@ -53,32 +52,4 @@ public GbmMojoModel(String[] columns, String[][] domains) {
         return score0(row, 0.0, preds);
     }
 
-    /**
```

Comment:
It took me a while to figure out what was going on in terms of the number of compressed trees per class.  Which is why I added this comment.

It looks like you are trying to unify the code in the shared class, which is good, but you should add a public accessor function with a good comment and make the actual field private.